rotkehlchen/api/v1/encoding.py

```diff
@@ -713,7 +713,7 @@ class NewUserSchema(BaseUserSchema):
 
 class AllBalancesQuerySchema(Schema):
     async_query = fields.Boolean(missing=False)
-    save_data = fields.Boolean(missing=True)
+    save_data = fields.Boolean(missing=None)
```

Comment:
missing should be `False` with the new changes, not `None` as explained in the other comment